libs/designsystem/src/lib/components/flag/flag.component.scss

```diff
@@ -22,19 +22,30 @@
   }
 }
 
-@each $color-name,
-  $color-value
-    in map.merge(
-      utils.$notification-colors,
-      (
-        'semi-light': utils.get-color('semi-light'),
-        'danger': utils.$danger-background-weak,
-      )
-    )
-{
+$_flag-notification-colors: map.merge(
+  utils.$notification-colors,
+  (
+    'semi-light': utils.get-color('semi-light'),
+  )
+);
+$_flag-notification-colors: map.remove($_flag-notification-colors, 'danger');
+
+@each $color-name, $color-value in $_flag-notification-colors {
```

Comment:
`$color-value` is no longer necessary here, right?

Reply:
no, you're right

Reply:
```suggestion
@each $color-name in $_flag-notification-colors {
```

Reply:
Just found out it is necessary, since `$_flag-notification-colors` is a map.

I'll provide a rewritten solution that uses a list of color names instead. We can either choose that solution or we can just keep the `@each $color-name, $color-value in $_flag-notification-colors` solution.